### DISCUSSION TITLE
清华大学魏杰教授关于当下宏观经济讲座

### DISCUSSION THREAD

**Discussion by hengbigbro** [RoadSideUniversity/ClubHouse2024#45] [General]
https://www.reddit.com/r/China_irl/comments/18aj2tl/%E5%88%86%E4%BA%AB%E4%B8%80%E7%AF%87%E6%B8%85%E5%8D%8E%E5%A4%A7%E5%AD%A6%E9%AD%8F%E6%9D%B0%E6%95%99%E6%8E%88%E5%85%B3%E4%BA%8E%E5%BD%93%E4%B8%8B%E5%AE%8F%E8%A7%82%E7%BB%8F%E6%B5%8E%E8%AE%B2%E5%BA%A7%E5%A2%99%E5%86%85%E5%B7%B2%E7%BB%8F%E5%B1%8F%E8%94%BD/

**Comment by Lewisrxliu** [comment]
干货！ 👍感谢分享

**Reply by yibingbingwu** [reply]
👍感谢分享! tldr: 经济上义无反顾地向市场经济转型。但做不到。政治改革是“纲举目张”

**Comment by Lewisrxliu** [comment]
北大萧国亮
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/9ca85796-49ae-4ff2-a922-e6fff4eb1e56)
